modules/post/windows/manage/priv_migrate.rb

```diff
@@ -60,7 +61,9 @@ def run
   def get_pid(proc_name)
     processes = client.sys.process.get_processes
     processes.each do |proc|
-      return proc['pid'] if proc['name'] == proc_name
+      if proc['name'] == proc_name
```

Comment:
You are correct, thank you for the suggestion.  The change will be applied shortly.